bin/cron/delete_accounts

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/db'
+require 'cdo/only_one'
+
+OPEN_ENDED_LEVEL_TYPES = %w(
+  Applab
+  FreeResponse
+  Gamelab
+  Weblab
+).freeze
```

Comment:
Most definitely. Amongst many other things...the exact list is still getting sorted out.